tools/src/main/scala/scala/scalanative/build/LLVM.scala

```diff
@@ -298,10 +298,22 @@ private[scalanative] object LLVM {
       buildTargetLinkOpts ++ flto ++ debugFlags ++ platformFlags ++ linkNameFlags ++ output ++ sanitizer ++ target
     }
     val paths = objectsPaths.map(_.abs)
+    val (lldStartOptions, lldEndOptions) =
+      if (config.linkingOptions.contains("-fuse-ld=lld")) {
```

Comment:
`-fuse-ld=lld` is also set internally in same cases on Windows.  We should check flags insteads

Reply:
I guess this is a problem we have in general if we use that linker, so we probably can add it in all cases where we see the flag. But I can add only when the user adds it manually if you prefer it.

Reply:
Is it just lld? Would that affect Mold as well? Situation would be different if this is a sign of a fundamental issue, not a peculiarity of some particular linker.